spyder/plugins/ipythonconsole/widgets/help.py

```diff
@@ -125,5 +129,6 @@ def _handle_inspect_reply(self, rep):
             if content.get('status') == 'ok' and content.get('found', False):
                 signature = self.get_signature(content)
                 if signature:
-                    self._control.show_calltip(_("Arguments"), signature,
-                                               signature=True, color='#2D62FF')
+                    # TODO: Pass the language from the Console to the calltip
+                    self._control.show_calltip(signature, color='#999999',
+                                               is_python=True)
```

Comment:
@ccordoba12 how can I get the information of the actual language the (Ipython) console is running. To make sure this is generic.

Reply:
https://github.com/spyder-ide/spyder/issues/9194